pkgs/development/compilers/cudatoolkit/hooks/mark-for-cudatoolkit-root-hook.sh

```diff
@@ -0,0 +1,10 @@
+# shellcheck shell=bash
+
+markForCUDAToolkit_ROOT() {
+    for outputIndex in "${!outputs[@]}"; do
+        mkdir -p "${!outputs[$outputIndex]}/nix-support"
+        touch "${!outputs[$outputIndex]}/nix-support/include-in-cudatoolkit-root"
+    done
```

Comment:
@ConnorBaker looks like the error comes from this bit. I think bash has just concatenated the output names (`(doc out lib)`) into a space-separated string

Reply:
Yup, `${outputs[$outputIndex]}` expands to `"doc out lib"`, idk hwy

Reply:
Looks like making it an array requires `structuredAttrs`...